.github/workflows/continous-delivery.yml

```
@@ -104,6 +104,17 @@ jobs:
         with:
           short-length: 7
 
+      # Makes the Zcash network name lowercase.
+      #
+      # Labels in GCP are required to be in lowercase, but the blockchain network
+      # uses sentence case, so we need to downcase ${{ inputs.network || 'Mainnet' }}.
+      #
+      # Passes the lowercase network to subsequent steps using $NETWORK env variable.
+      - name: Downcase network name for labels
+        run: |
+          NETWORK_CAPS="${{ inputs.network || 'Mainnet' }}"
```

I'm not sure if the second `|| 'Mainnet'` is needed

```suggestion
          NETWORK_CAPS="${{ inputs.network }}"
```